Submitted By:            Armin K. <krejzi at email dot com>
Date:                    2014-08-23
Initial Package Version: 13.0.0
Upstream Status:         Fixed
Origin:                  Upstream
Description:             Fixes building with Glibc 2.20

--- a/shared/vmmouse_client.c	2013-01-15 19:36:50.000000000 +0100
+++ b/shared/vmmouse_client.c	2014-08-23 15:13:54.336165103 +0200
@@ -35,6 +35,10 @@
  *
  */
 
+#ifdef HAVE_CONFIG_H
+#include "config.h"
+#endif
+
 #include "vmmouse_client.h"
 #include "vmmouse_proto.h"
 
--- a/shared/vmmouse_proto.c	2013-01-15 19:36:50.000000000 +0100
+++ b/shared/vmmouse_proto.c	2014-08-23 15:13:54.336165103 +0200
@@ -33,6 +33,10 @@
  */
 
 
+#ifdef HAVE_CONFIG_H
+#include "config.h"
+#endif
+
 #include "vmmouse_proto.h"
 
 
--- a/shared/vmmouse_proto.h	2013-01-15 19:36:50.000000000 +0100
+++ b/shared/vmmouse_proto.h	2014-08-23 15:13:54.336165103 +0200
@@ -36,11 +36,6 @@
 #ifndef _VMMOUSE_PROTO_H_
 #define _VMMOUSE_PROTO_H_
 
-
-#ifdef HAVE_CONFIG_H
-#include "config.h"
-#endif
-
 #include <stdint.h>
 
 #ifdef HAVE_XORG_SERVER_1_1_0
--- a/tools/vmmouse_detect.c	2013-01-15 19:36:50.000000000 +0100
+++ b/tools/vmmouse_detect.c	2014-08-23 15:13:54.336165103 +0200
@@ -26,14 +26,14 @@
  */
 
 
-#include <stdlib.h>
-#include <signal.h>
-#include "vmmouse_client.h"
-
 #ifdef HAVE_CONFIG_H
 #include "config.h"
 #endif
 
+#include <stdlib.h>
+#include <signal.h>
+#include "vmmouse_client.h"
+
 void
 segvCB(int sig)
 {
